cmd/up.go

```diff
@@ -18,5 +18,12 @@ func (h *Handler) Up(ctx context.Context, req *entity.CommandRequest) error {
 	} else {
 		ui.StopSpinner(fmt.Sprintf("☁️ Deploy available at %s\n", ui.GrayText(url)))
 	}
-	return nil
+	detach, err := req.Cmd.Flags().GetBool("detach")
+	if err != nil {
+		return err
+	}
+	if detach {
+		return nil
+	}
+	return h.ctrl.GetActiveDeploymentLogs(ctx, 0)
```

Comment:
Does this mean that `railway up`, by default, streams both build and runtime logs indefinitely after?

Reply:
Yes it does

Reply:
I think I'd vote to only see build logs after `up`. 

My flow is usually to make sure the deploy worked, then go back to writing code. Right now it's difficult to see when it switches from build to deploy then you have to ctrl-C out (which may also not be obvious to less-experienced devs). 

One idea might be to detect when the build succeeded and prompt the user to continue streaming the deploy logs or exit.

Reply:
I figured the "Build Completed" line is solid for this

My issue is that we fire build completed regardless of outcome, and only fetch the actual deployment logs (Which contains the "Failed to Deploy!" stuff. If we do a prompt, users might not see that

Could see it either way tho. Don't really have a strong opinion so can add a prompt pretty easily